src/psyclone/parse.py

```diff
@@ -1,7 +1,8 @@
 # -----------------------------------------------------------------------------
 # BSD 3-Clause License
 #
-# Modifications copyright (c) 2017, Science and Technology Facilities Council
+# Modifications copyright (c) 2017-2018, Science and Technology
```

Comment:
Actually I think Imperial agreed to pass copyright to STFC, but we obviouly keep authorship information.

Reply:
Done.